packages/integration-tests/src/tests/ui/smoke.test.ts

```diff
@@ -5,10 +5,6 @@ describe('smoke testing', () => {
   const consoleUsername = 'admin';
   const consolePassword = generatePassword();
 
-  beforeEach(async () => {
-    await page.waitForTimeout(1000);
-  });
-
```

Comment:
should we just keep this?

Reply:
No, let's remove it, for it will increase the possibility of test failure IMO

Reply:
After further investigation, I think this doesn't work as expected, either. It makes no difference to the current issue